tests/integration/targets/gitlab_project_variable/tasks/main.yml

```diff
@@ -218,6 +218,22 @@
     that:
       - gitlab_project_variable_state is not changed
 
+- name: change environment scope
+  gitlab_project_variable:
+    api_url: "{{ gitlab_host }}"
+    api_token: "{{ gitlab_login_token }}"
+    project: "{{ gitlab_project_name }}"
+    vars:
+      ACCESS_KEY_ID:
+        environment_scope: testing
+        
```

Comment:
can you remove the blank line between task and register option @castorinop?  
furthermore, can you run the test twice, to check if a second call results in a `is not changed`?

Reply:
done!